### PR TITLE
Full NASK extension - supports all NASK extensions

### PR DESCRIPTION
**Pull request by johnny-bit**
This adds full NASK extensions. NASK is Polish domain registry
To connect to NASK you need to use HTTPClient as NASK uses EPP over HTTPS

### CODE REVIEW

**Comment by johnny-bit** [comment]
Squashed and ready to merge. should be no conflict with #65

File: src/AfriCC/EPP/AbstractFrame.php [hunk 1]
```diff
@@ -220,7 +228,7 @@ protected function realxpath($path)
             array_unshift($path_parts, $this->mapping . ':' . $this->command);
         }
 
-        if (!empty($this->command)) {
+        if (!empty($this->command) && !$this->ignore_command) {
```
Comment (johnny-bit) [inline src/AfriCC/EPP/AbstractFrame.php:231]: This was required in order to succesfuly build NASKs `Report` suite if you're wondering ;)

File: src/AfriCC/EPP/Extension/NASK/ObjectSpec.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    public static $serviceExtensions = [
+        'http://www.dns.pl/nask-epp-schema/extcon-2.0',
+        'http://www.dns.pl/nask-epp-schema/extdom-2.0',
+        'http://www.dns.pl/nask-epp-schema/secDNS-2.0',
+    ];
+
+    private static $backup;
+
+    public static function overwriteParent()
```
Comment (johnny-bit) [inline src/AfriCC/EPP/Extension/NASK/ObjectSpec.php:46]: Maybe there's a better way to overwrite `$services` and `$serviceExtensions` in `ObjectSpec` when dealing with NASKs own... But I haven't got better idea :cry:

File: src/AfriCC/EPP/ObjectSpec.php [hunk 3]
```diff
@@ -40,10 +40,10 @@ class ObjectSpec
 
     public static function xmlns($object)
     {
-        if (!isset(self::$specs[$object]['xmlns'])) {
+        if (!isset(static::$specs[$object]['xmlns'])) {
```
Comment (johnny-bit) [inline src/AfriCC/EPP/ObjectSpec.php:43]: This was required to even be able to refer to specs in child classes.

**Review by lifeofguenter — APPROVED** [review]

**Pull request merged**